dev-docs/source/DeveloperAccounts.rst

```diff
@@ -0,0 +1,63 @@
+.. _DeveloperAccounts:
+
+==================
+Developer Accounts
+==================
+
+.. contents::
+  :local:
+
+User Names
+----------
+
+Simple, easy to recognise user names are preferred. For example "Nick Draper", or if spaces are not allowed "NickDraper".
+
+Account Creation
+----------------
+
+- Create a **Mantid Wiki** account; follow `this link <https://www.mantidproject.org/Special:RequestAccount>`_.
+- Sign up to the **Mantid Developers** email list; follow `this link <http://lists.mantidproject.org/mailman/listinfo/mantid-developers>`_.
+- Sign up for the **Mantid Slack** channel; follow `this link <https://mantid.slack.com/>`_.
+- If you don't already have one, sign up for a **Github** account; follow `this link <https://github.com/>`_.
+	+ Remember that your username should be easily identifiable.
+	+ Contact one of the "Owners" on `this list <https://github.com/orgs/mantidproject/people>`_ to add you to the developer team.
```

Comment:
You can use this link to prefill the "role:owner" search: https://github.com/orgs/mantidproject/people?query=role%3Aowner

Reply:
Nice.